.github/ISSUE_TEMPLATE.md

```diff
@@ -1,14 +1,29 @@
-* xeus-cookiecutter version:
+# Checklist
+Before submitting an issue please confirm you have checked the follow
+- [ ] This issue is not a duplicate of one already open
+- [ ] The title of the issue is clear
+
+# System information
+* xeus-cpp version:
 * Operating System:
+* CPU architechture:
+
+## Description of issue
+Please tick the boxes which apply to your issue
+- [ ] Bug in xeus-cpp
+- [ ] Request for new feature to be added
+- [ ] Issue with documentation
+- [ ] Other (please state below)
```

Comment:
This should be handled by GH label, when we do triage. Some of the previous sections may not apply to all the possibilities here. Also the other sections make me think that this template is for bug reports.